src/brucke_config.erl

```diff
@@ -1,5 +1,5 @@
 %%%
-%%%   Copyright (c) 2016 Klarna AB
+%%%   Copyright (c) 2016-2017 Klarna AB
```

Comment:
never mind, no big deal.
but please keep in mind as i also commented earlier: we don't update a file only for the copyright notice years.